_faq/bugs-play-downloaded-videos.html

```diff
@@ -0,0 +1,17 @@
+---
+categories:
+- player
+- bugs
+type: info
+title: "Playing downloaded videos"
+---
+<section>
+    <p>To play downloaded videos a external video player is required.</p>
+    <p>There have been considerations implementing this directly into NewPipe however this was dropped due to a bunch of reasons:</p>
+    <ul>
+        <li>Normally a phone is shipped with a video player.</li>
+        <li>NewPipe is not designed to be an all-around video player.</li>
+        <li>A huge addtional maintenance effort would be required to make this work.</li>
+        <li>There are already <a href="https://search.f-droid.org/?q=video+player">entire apps</a> available that deal with offline playback.</li>
```

Comment:
```suggestion
        <li>There are other <a href="https://search.f-droid.org/?q=video+player">apps</a> available that deal with offline playback, in case the phone doesn't have one.</li>
```